nss-day-cohort-50/Class-Site
# fixed background not responding on mobile

## Description of Proposed Changes
Fixed background being unresponsive when viewing on mobile.
- 

## Steps to Test
Open dev tools and check that background responds to various mobile views 

Outline the steps to test

```sh
git fetch --all
git checkout th-mediaBackgroundFix

```


## Impacted Areas in Application

css

* 

## Mentions @pulley871 

Tag users that need to review this code
